styles/05-objects/card.css

```diff
@@ -0,0 +1,148 @@
+/* .cards, */
+
+.card-container,
+.card-primary-content,
+.actions {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.actions {
+  flex-wrap: nowrap;
+}
+
+.cards {
+  padding: 10px;
+  justify-content: space-around;
+  gap: 4rem;
+}
+
+.card-container {
+  background-color: white;
```

Comment:
since you are using #fff below, keep i consistent use that here too

Reply:
Thanks for pointing it out